setup-vcenter-vms.yml

```diff
@@ -374,3 +374,55 @@
       when: enable_ocs_storage|bool == True
       tags:
         - "5"
+
+    - name: VMSETUP 5.2 | Label Core Cluster VM's
+      command: "{{ govc_file }} vm.change -vm={{ item.name }} -annotation=Cluster-Name-{{ BUILD_LAB }}-{{ config.clusterName }}-{{ domain }},OCP-Release-{{ ocp_version }},Instance-Owner-Red-Hat-SSA-Team"
+      environment:
+        GOVC_HOST: "{{ vcenter.ip }}"
+        GOVC_URL: "https://{{ vcenter.ip }}/sdk"
+        GOVC_USERNAME: "{{ vcenter.username }}"
+        GOVC_PASSWORD: "{{ vcenter.password }}"
+        GOVC_INSECURE: 1
+        GOVC_DATASTORE: "{{ vcenter.datastore }}"
+        GOVC_DATACENTER: "{{ vcenter.datacenter }}"
+        GOVC_CLUSTER: "{{ vcenter.cluster }}"
+        GOVC_NETWORK: "{{ vcenter.vm_network_1 }}"
+      loop:
+         - { name : "bootstrap" }
+         - { name : "master0" }
+         - { name : "master1" }
+         - { name : "master2" }
+         - { name : "worker0" }
+         - { name : "worker1" }
+         - { name : "worker2" }
+      retries: 3
+      delay: 3
+      register: result
+      until: result.rc == 0
```

Comment:
I don't understand the purpose. It will loop all the nodes and if some of them fail, it will fail the whole task.

Reply:
Sure, I will remove the retries